client/app/components/compose.coffee

```diff
@@ -52,6 +52,10 @@ module.exports = Compose = React.createClass
 
 
     shouldComponentUpdate: (nextProps, nextState) ->
+        if @state.saving isnt nextState.saving
```

Comment:
I think, you should probably do it there : 
https://github.com/cozy/cozy-emails/blob/f490f6139aa177d490be8ac98f767ec7d4a8df4b/client/app/components/compose.coffee#L459